src/hearings/converters/additional-instructions.converter.spec.ts

```diff
@@ -0,0 +1,23 @@
+import {cold} from 'jasmine-marbles';
+import {of} from 'rxjs';
+import {initialState} from '../hearing.test.data';
+import {State} from '../store';
+import {AdditionalInstructionsAnswerConverter} from './additional-instructions.answer.converter';
+
+describe('AdditionalInstructionsAnswerConverter', () => {
```

Comment:
probably this file needs to rename to additional-instructions.answer.converter.spec.ts?